Add caution message to acu.md (#123278)
Azure/azure-content: articles/virtual-machines/acu.md

**Applies to:** :heavy_check_mark: Linux VMs :heavy_check_mark: Windows VMs :heavy_check_mark: Flexible scale sets :heavy_check_mark: Uniform scale sets

> [!CAUTION]

> > This page is no longer being maintained and will be archived 12/16/2024.

Reviewer: ```suggestion
> This page is no longer being maintained and is planned for archival on 12/16/2024.
```
mitigating future promise.
